### PR TITLE
Appium graduation submission

### PR DESCRIPTION
**Pull request by kborchers**

### CODE REVIEW

File: Projects/Appium.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Project description
+
+Appium is an open-source tool for automating native, mobile web, and hybrid applications on iOS and Android platforms. Written in Node.js, Appium is an implementation of the standard WebDriver protocol, allowing client access from any programming language (not just JS). Appium also allows cross-platform automation: it allows you to write tests against multiple platforms (iOS, Android, Windows), using the same API. This enables code reuse between iOS, Android, and Windows testsuites. Appiums vision is to expand itself as an umbrella protocol for even more kinds of automation, from desktop apps to IoT devices to VR experiences.
+
```
Comment (agcolom) [inline Projects/Appium.md]: Appiums vision should be Appium's vision?

File: Projects/Appium.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Project description
+
+Appium is an open-source tool for automating native, mobile web, and hybrid applications on iOS and Android platforms. Written in Node.js, Appium is an implementation of the standard WebDriver protocol, allowing client access from any programming language (not just JS). Appium also allows cross-platform automation: it allows you to write tests against multiple platforms (iOS, Android, Windows), using the same API. This enables code reuse between iOS, Android, and Windows testsuites. Appiums vision is to expand itself as an umbrella protocol for even more kinds of automation, from desktop apps to IoT devices to VR experiences.
+ 
+# Project history
+
+Appium has been around in one form or another since 2012. Its been under the direction of various individuals and organizations, and its even been implemented in 3 different programming languages! The full story is at [http://appium.io/history.html](http://appium.io/history.html)
```
Comment (agcolom) [inline Projects/Appium.md]: Its been -> It's been 
its even been -> it's even been

File: Projects/Appium.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+* TSC Member Count: 5
+* Twitter followers: 3.1k (@AppiumDevs)
+* GitHub stars: 5.7k
+* High-profile Dependents: Lots of products and services, but no high profile code dependents in the OSS world (Appium is more of an end user tool; companies and projects use it in their test pipeline. For example, [React Native](https://github.com/facebook/react-native/blob/master/scripts/run-ci-e2e-tests.js)).
+
+# Project scope
+
+Appium was designed to meet mobile automation needs according to a philosophy outlined by the following four tenets:
+
+1. You shouldnt have to recompile your app or modify it in any way in order to automate it.
```
Comment (agcolom) [inline Projects/Appium.md]: You shouldnt -> You shouldn't

File: Projects/Appium.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+2. You shouldnt be locked into a specific language or framework to write and run your tests.
```
Comment (agcolom) [inline Projects/Appium.md]: You shouldnt -> You shouldn't

File: Projects/Appium.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+3. A mobile automation framework shouldnt reinvent the wheel when it comes to automation APIs.
```
Comment (agcolom) [inline Projects/Appium.md]: shouldnt -> shouldn't

File: Projects/Appium.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+4. A mobile automation framework should be open source, in spirit and practice as well as in name!
+
+Appium is now a tight federation of drivers which enable automation via the WebDriver protocol for specific platforms. Appiums vision is for this federation to grow, extending the scope of the original Selenium project far beyond the edge of the browser.
```
Comment (agcolom) [inline Projects/Appium.md]: Appiums vision -> Appium's vision

**Comment by kborchers** [comment]
Updated. Thanks @agcolom!

**Pull request closed without merging**